### PR TITLE
Update node-js-sdk.md

### PR DESCRIPTION
**Pull request by jwulf**
A customer reached out to support because they used the Docker profile with no Identity and the connection fails because we assume OAuth.

This patch informs the users how to turn OAuth off.

## Description

Ref: https://jira.camunda.com/browse/SUPPORT-22663

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ X ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ X ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ X ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by conceptualshark — DISMISSED** [review]

**Review by pepopowitz — CHANGES_REQUESTED** [review]
This looks good to me, except that I think the bot requesting corresponding changes in `docs/` is accurate. We should apply the update to version next, in addition to 8.5, to carry it forward into new releases.

**Comment by jwulf** [comment]
> This looks good to me, except that I think the bot requesting corresponding changes in `docs/` is accurate. We should apply the update to version next, in addition to 8.5, to carry it forward into new releases.

Is that https://github.com/camunda/camunda-docs/pull/4027?

**Comment by pepopowitz** [comment]
> > This looks good to me, except that I think the bot requesting corresponding changes in `docs/` is accurate. We should apply the update to version next, in addition to 8.5, to carry it forward into new releases.
> 
> Is that #4027?

Yes! Sorry, I didn't realize there was a second associated PR.

**Review by pepopowitz — APPROVED** [review]

File: versioned_docs/version-8.5/apis-tools/node-js-sdk.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ If your installation does not have TLS on Zeebe:
 export CAMUNDA_SECURE_CONNECTION=false
 ```
 
+If you are using the "basic components" (no Identity), you need to disable OAuth:
```
[the remark this answers is not in the document]
Comment (akeller) [reply versioned_docs/version-8.5/apis-tools/node-js-sdk.md:72]: @christinaausley Josh will be out for the week on FTO. Can you discuss with @conceptualshark if you'd like to make this change?

**Review by christinaausley — APPROVED** [review]

**Pull request merged**